docs/guides/setting-url-wordpress-source-data..md

```diff
@@ -0,0 +1,114 @@
+# Setting the URL of the WordPress source of data
+
+The most important setting in a Frontity project is the WordPress installation that can be used as the source of data
+
+The format of the WordPress URL will be different depending on the type of WordPress installation used with the Frontity project so the type of WordPress installation may require different ways of setting this url in the Frontity settings
```

Comment:
Replace
"The format of the WordPress URL will be different depending on the type of WordPress installation used with the Frontity project so the type of WordPress installation may require different ways of setting this url in the Frontity settings"
with
"The format of the WordPress URL varies depending on the type of WordPress installation used as the data source for the Frontity project, so the type of WordPress installation determines how this the URL should be set in the Frontity configuration file `frontity.settings.js`"